documentation-site/components/yard/yard.tsx

```diff
@@ -314,6 +320,9 @@ const Yard: React.FC<
             updateCode(dispatch, formatCode(state.code));
           }}
         >
+          <MdFormatIndentIncrease
+            style={{paddingRight: theme.sizing.scale100}}
+          />{' '}
           Format
```

Comment:
Did you mean to remove the button text? If so, could you add a tooltip instead?

Reply:
Currently, it doesn't remove the text, just adds the icon - in L326, you can see the "Format" text. Is that what you meant? Maybe I misunderstood

Reply:
The screenshot is confusing then since 3 first buttons only contain icons and no text. But preview (and code) has text next to icons for those 3 buttons.

Reply:
oh, I forgot to update those - sorry about that